caluma/data/form-config.json

```diff
@@ -533,178 +563,177 @@
         "source": null
     }
 },
 {
     "model": "caluma_form.option",
-    "pk": "review-document-decision-continue",
+    "pk": "review-document-decision-",
     "fields": {
-        "created_at": "2021-05-31T09:54:31.456Z",
-        "modified_at": "2021-06-03T09:17:44.755Z",
+        "created_at": "2021-05-31T08:31:45.938Z",
+        "modified_at": "2021-05-31T08:32:03.185Z",
         "created_by_user": null,
         "created_by_group": null,
         "modified_by_user": null,
         "modified_by_group": null,
-        "label": "{\"en\": null, \"de\": \"Weiterf\\u00fchrung\", \"fr\": null}",
+        "label": "{\"de\": \"Abschluss\", \"fr\": null, \"en\": null}",
         "is_archived": false,
         "meta": {},
         "source": null
     }
 },
 {
     "model": "caluma_form.option",
-    "pk": "review-document-decision-reject",
+    "pk": "review-document-decision-complete",
     "fields": {
-        "created_at": "2021-05-31T09:41:37.580Z",
-        "modified_at": "2021-06-03T09:17:44.758Z",
-        "created_by_user":  null,
+        "created_at": "2021-05-31T09:54:31.474Z",
+        "modified_at": "2021-06-03T09:17:44.753Z",
+        "created_by_user": null,
         "created_by_group": null,
         "modified_by_user": null,
         "modified_by_group": null,
-        "label": "{\"en\": null, \"de\": \"R\\u00fcckweisung\", \"fr\": null}",
+        "label": "{\"de\": \"Abschluss\", \"fr\": null, \"en\": null}",
         "is_archived": false,
         "meta": {},
         "source": null
     }
 },
 {
     "model": "caluma_form.option",
-    "pk": "circulation-decision-approved",
+    "pk": "review-document-decision-continue",
     "fields": {
-        "created_at": "2021-06-25T13:19:30.349Z",
-        "modified_at": "2021-06-25T13:19:30.349Z",
+        "created_at": "2021-05-31T09:54:31.456Z",
+        "modified_at": "2021-06-03T09:17:44.755Z",
         "created_by_user": null,
         "created_by_group": null,
         "modified_by_user": null,
         "modified_by_group": null,
-        "label": "{\"en\": null, \"de\": \"Genehmigt\", \"fr\": null}",
+        "label": "{\"de\": \"Weiterf\\u00fchrung\", \"fr\": null, \"en\": null}",
         "is_archived": false,
         "meta": {},
         "source": null
     }
 },
 {
     "model": "caluma_form.option",
-    "pk": "circulation-decision-declined",
+    "pk": "review-document-decision-reject",
     "fields": {
-        "created_at": "2021-06-25T13:19:30.354Z",
-        "modified_at": "2021-06-25T13:19:30.354Z",
+        "created_at": "2021-05-31T09:41:37.580Z",
+        "modified_at": "2021-06-03T09:17:44.758Z",
         "created_by_user": null,
         "created_by_group": null,
         "modified_by_user": null,
         "modified_by_group": null,
-        "label": "{\"en\": null, \"de\": \"Abgelehnt\", \"fr\": null}",
+        "label": "{\"de\": \"R\\u00fcckweisung\", \"fr\": null, \"en\": null}",
         "is_archived": false,
         "meta": {},
         "source": null
```

Comment:
@open-dynaMIX I think we still need sorting of the config files to avoid large diffs...